### PR TITLE
sonar 5.0.1

### PR DESCRIPTION
**Pull request by a1dutch**

### CODE REVIEW

**Comment by a1dutch** [comment]
added test and cleaned up the audit checks reported

File: Library/Formula/sonar.rb [hunk 1]
```diff
@@ -46,4 +44,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    assert_match /SonarQube/, pipe_output("#{bin}/sonar status")
```
Comment (mikemcquaid) [inline Library/Formula/sonar.rb]: Use `shell_output` here.

Comment (DomT4) [inline Library/Formula/sonar.rb]: Apologies, this came from me. ` sonar status ` doesn't seem to exit by itself, or at least not for a prolonged period, so I didn't know if that would impact ` shell_output ` in the same manner.

Comment (a1dutch) [inline Library/Formula/sonar.rb]: I had some errors when trying to get the sonar4 test working in
homebrew-versions this is a direct copy of the test implemented there.

I will take a look again, trying to keep the scripts the same as much as
possible.

On 1 Mar 2015, at 21:47, Mike McQuaid <notifications@github.com> wrote:

In Library/Formula/sonar.rb
<https://github.com/Homebrew/homebrew/pull/37237#discussion_r25571910>:

> @@ -46,4 +44,8 @@ def plist; <<-EOS.undent
>      </plist>
>      EOS
>    end
> +
> +  test do
> +    assert_match /SonarQube/, pipe_output("#{bin}/sonar status")

Use shell_output here.

—
Reply to this email directly or view it on GitHub
<https://github.com/Homebrew/homebrew/pull/37237/files#r25571910>.

**Comment by a1dutch** [comment]
changing to `assert_match /SonarQube/, shell_output("#{bin}/sonar status")` results in `Error: sonar: failed <0> expected but was <1>.`

**Comment by mikemcquaid** [comment]
Pass `1` as the second argument as it's checking the expected return code. Thanks!

**Comment by a1dutch** [comment]
hopefully that should do it

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**